diff --git a/toolkit/library/libxul.symbols b/toolkit/library/libxul.symbols
index e63e3e74426b..57337d0858de 100644
--- a/toolkit/library/libxul.symbols
+++ b/toolkit/library/libxul.symbols
@@ -3,6 +3,19 @@ uprofiler_get
 #ifdef XP_IOS
 MainProcessInit
 ChildProcessInit
+ReportJITStatusForChild
+JITListeningSecret
+GeckoViewSetBackingScaleOverride
+ReynardAVPlayerNotifyMetadata
+ReynardAVPlayerNotifyFrame
+ReynardAVPlayerNotifyTime
+ReynardAVPlayerNotifyEnded
+ReynardAVPlayerNotifyError
+ReynardAVPlayerNotifyProtected
+ReynardAVPlayerNotifyHostState
+ReynardFairPlayNotifyEncrypted
+ReynardFairPlayNotifyKeyMessage
+ReynardFairPlayNotifyParserKeyMessage
 GeckoViewOpenWindow
 #endif
 #ifdef ANDROID
