diff --git a/modules/woff2/moz.build b/modules/woff2/moz.build
index 69b3e1df251f..dd0627ece7f1 100644
--- a/modules/woff2/moz.build
+++ b/modules/woff2/moz.build
@@ -15,7 +15,8 @@ EXPORTS.woff2 += [
 
 ]
 
-if not CONFIG["MOZ_WASM_SANDBOXING_WOFF2"]:
+if not CONFIG["MOZ_WASM_SANDBOXING_WOFF2"] or \
+    CONFIG["MOZ_WIDGET_TOOLKIT"] == "uikit":
     UNIFIED_SOURCES += woff2_sources
 
     # We allow warnings for third-party code that can be updated from upstream.
