diff --git a/gfx/ipc/RemoteCompositorSession.cpp b/gfx/ipc/RemoteCompositorSession.cpp
index 5dbdea7f3eeb..b61b20e8442a 100644
--- a/gfx/ipc/RemoteCompositorSession.cpp
+++ b/gfx/ipc/RemoteCompositorSession.cpp
@@ -11,9 +11,9 @@
 #include "mozilla/layers/CompositorBridgeChild.h"
 #include "mozilla/layers/GeckoContentController.h"
 #include "nsIWidget.h"
-#if defined(MOZ_WIDGET_ANDROID)
+#if defined(MOZ_WIDGET_ANDROID) || defined(MOZ_WIDGET_UIKIT)
 #  include "mozilla/layers/UiCompositorControllerChild.h"
-#endif  // defined(MOZ_WIDGET_ANDROID)
+#endif
 
 namespace mozilla {
 namespace layers {
@@ -37,9 +37,9 @@ RemoteCompositorSession::RemoteCompositorSession(
 RemoteCompositorSession::~RemoteCompositorSession() {
   // This should have been shutdown first.
   MOZ_ASSERT(!mCompositorBridgeChild);
-#if defined(MOZ_WIDGET_ANDROID)
+#if defined(MOZ_WIDGET_ANDROID) || defined(MOZ_WIDGET_UIKIT)
   MOZ_ASSERT(!mUiCompositorControllerChild);
-#endif  // defined(MOZ_WIDGET_ANDROID)
+#endif
 }
 
 void RemoteCompositorSession::NotifySessionLost() {
@@ -85,12 +85,12 @@ void RemoteCompositorSession::Shutdown() {
   }
   mCompositorWidgetDelegate = nullptr;
   mWidget = nullptr;
-#if defined(MOZ_WIDGET_ANDROID)
+#if defined(MOZ_WIDGET_ANDROID) || defined(MOZ_WIDGET_UIKIT)
   if (mUiCompositorControllerChild) {
     mUiCompositorControllerChild->Destroy();
     mUiCompositorControllerChild = nullptr;
   }
-#endif  // defined(MOZ_WIDGET_ANDROID)
+#endif
   GPUProcessManager::Get()->UnregisterRemoteProcessSession(this);
 }
 
