diff --git a/js/src/gc/Zone.cpp b/js/src/gc/Zone.cpp
index ed2d6124c4b8..cd8f62b79862 100644
--- a/js/src/gc/Zone.cpp
+++ b/js/src/gc/Zone.cpp
@@ -16,6 +16,7 @@
 #include "jit/Invalidation.h"
 #include "jit/JitScript.h"
 #include "jit/JitZone.h"
+#include "jit/ProcessExecutableMemory.h"
 #include "vm/Runtime.h"
 #include "vm/Time.h"
 
@@ -35,7 +36,7 @@ Zone* const Zone::NotOnList = reinterpret_cast<Zone*>(1);
 
 ZoneAllocator::ZoneAllocator(JSRuntime* rt, Kind kind)
     : JS::shadow::Zone(rt, rt->gc.marker().tracer(), kind),
-      jitHeapThreshold(size_t(jit::MaxCodeBytesPerProcess * 0.8)) {}
+      jitHeapThreshold(size_t(jit::ReservedCodeBytesPerProcess() * 0.8)) {}
 
 ZoneAllocator::~ZoneAllocator() {
 #ifdef DEBUG
